provider/src/main/java/com/cloudera/director/google/sql/GoogleCloudSQLInstance.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2015 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.director.google.sql;
+
+import com.cloudera.director.spi.v1.database.util.AbstractDatabaseServerInstance;
+import com.cloudera.director.spi.v1.model.DisplayProperty;
+import com.cloudera.director.spi.v1.model.DisplayPropertyToken;
+import com.cloudera.director.spi.v1.model.util.SimpleDisplayPropertyBuilder;
+import com.cloudera.director.spi.v1.util.DisplayPropertiesUtil;
+import com.google.api.services.sqladmin.model.DatabaseInstance;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Maps;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+import java.util.Map;
+
+public class GoogleCloudSQLInstance
+    extends AbstractDatabaseServerInstance<GoogleCloudSQLInstanceTemplate, DatabaseInstance> {
+
+  private static final Logger LOG = LoggerFactory.getLogger(GoogleCloudSQLInstance.class);
+
+  /**
+   * The list of display properties (including inherited properties).
+   */
+  private static final List<DisplayProperty> DISPLAY_PROPERTIES =
+      DisplayPropertiesUtil.asDisplayPropertyList(GoogleSQLInstanceDisplayPropertyToken.values());
+
+  /**
+   * Returns the list of display properties for a Google Cloud SQL instance, including inherited properties.
+   *
+   * @return the list of display properties for a Google Cloud SQL instance, including inherited properties
```

Comment:
This duplicates the entire comment for the function. Does it make sense to do so? Should this just be shortened, given that this is just an accessor method?